lib/extensions/simple_form/components/html5_no_aria_required.rb

```diff
@@ -0,0 +1,15 @@
+module SimpleForm
+  module Components
+    module HTML5NoAriaRequired
+      include HTML5
+
+      def html5_no_aria_required(_wrapper_options)
```

Comment:
`@html5 = true`

Does that need to be added to the new method? It's in the original.

Reply:
@aduth that seemed to fix the mistyping of the number field. The inherited module initializes as false then gets set to true in the method that's being altered with html5_no_aria.